specs/interop/supervisor.md

```diff
@@ -37,32 +37,32 @@
       - [Access-list execution context](#access-list-execution-context)
       - [Access-list checks](#access-list-checks)
       - [`supervisor_checkAccessList` contents](#supervisor_checkaccesslist-contents)
-  - [Errors](#errors)
+  - [Data Availability Errors](#data-availability-errors)
```

Comment:
why the rename to "data availability errors"? are they all DA errors?

Reply:
they should all be, there is another issue open for removing the 'uninitialised chain database' error, since that's not interesting for a program depending on the supervisor to record with metrics and eventually potentially implement a recovery mechanism for.